lib/content-types/only.js

```diff
@@ -16,16 +16,32 @@ const only = (type, config) => {
         return attr;
       }
 
-      // Merge configuration for the inputs and the configuration
-      attr.inputs = merge(attr.inputs, config[attr.id]);
-
-      // Override the validation, type, and ID for all inputs that matter
-      inputs.forEach(input => {
-        attr.inputs[input].validation = attribute.inputs[input].validation;
-        attr.inputs[input].type = attribute.inputs[input].type;
-        attr.inputs[input].id = attribute.inputs[input].id;
-        attr.inputs[input].name = attribute.inputs[input].name;
-      });
+      // Flattens multiple instances
+      if (Array.isArray(config[attr.id])) {
+        const instance = [];
+        config[attr.id].forEach((data, index) => {
```

Comment:
Can you write this as a `.map` instead of a `forEach`? That'll simplify this code a lot and remove your need to add the index explicitly